core-api/src/utils/httpStatus.ts

```diff
@@ -1,14 +1,14 @@
 /**
- * @namespace Utils
+ * @namespace enum
```

Comment:
I believe @namespace is Utils, not sure.

Reply:
I suppose it's really `Utils`.

Reply:
Done